### PR TITLE
Added hyperbolic sin and cos to keras backend

### PR DESCRIPTION
**Pull request by KOLANICH**

### CODE REVIEW

File: tensorflow/python/keras/backend.py [hunk 1]
```diff
@@ -2107,6 +2107,32 @@ def minimum(x, y):
...
+def cosh(x):
+  """Computes cosh of x element-wise.
+
+  Arguments:
+      x: Tensor or variable.
+
+  Returns:
+      A tensor.
+  """
+  return math_ops.cosh(x)
```
Comment (arp95) [inline tensorflow/python/keras/backend.py]: LGTM @KOLANICH

**Review by arp95 — APPROVED** [review]
LGTM @KOLANICH

**Comment by KOLANICH** [comment]
Sorry, no CLAs. The changes are trivial. Feel free to recreate from your acc. Also we need `real` and `imag`

**Comment by kyscg** [comment]
Thanks, I'm on it. Can we close this? @KOLANICH

**Comment by KOLANICH** [comment]
@kyscg , are you going to create a new PR with the stuff?

**Comment by kyscg** [comment]
Yes, I will do it soon. Is there anything else you want me to keep in mind???

**Comment by KOLANICH** [comment]
`linalg.einsum` though I am not sure that it is avilable in other libs (it is unavailable in CNTK) and  `K.eye`'s `batch_shape` parameter.

**Pull request closed without merging**